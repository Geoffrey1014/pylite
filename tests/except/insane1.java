class Return2 {
    public static void main(String[] args) throws Exception
    {
        try {
            throw new Exception("Exception");
        } 
        finally {
            System.out.println("finally");
            return; /* suppress the exception */
        }
    }
}
