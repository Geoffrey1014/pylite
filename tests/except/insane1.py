def main():
    try:
        print 'try'
        0/0
    finally:
        print 'finally' 
        return  # don't rethrow the exception

print 'global start'
main()
print 'global end'
