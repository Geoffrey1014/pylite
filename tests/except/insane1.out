global start
try
finally
global end
